### PR TITLE
add client version message in connect response for gateway

### PR DESCRIPTION
**Pull request by 1uvu**

### CODE REVIEW

File: ccore/nebula/gateway/dao/dao.go [hunk 1]
```diff
@@ -276,10 +276,10 @@ func getMapInfo(valWarp *wrapper.ValueWrapper, _verticesParsedList *list, _edges
 }
 
 // Connect return if the nebula connect succeed
-func Connect(address string, port int, username string, password string, version nebula.Version) (nsid string, err error) {
-	nsid, err = pool.NewClient(address, port, username, password, version)
+func Connect(address string, port int, username string, password string, version nebula.Version) (nsid string, ver nebula.Version, err error) {
```
Comment (veezhang) [inline ccore/nebula/gateway/dao/dao.go:279]: How about just return `nebula.GraphClient, error`?

Comment (1uvu) [reply ccore/nebula/gateway/dao/dao.go:279]: Yes, ok.

File: controllers/db.go [hunk 2]
```diff
@@ -59,7 +59,7 @@ func (this *DatabaseController) Connect() {
 		this.Ctx.SetCookie("SameSite", "Strict")
 		this.SetSession(beego.AppConfig.String("sessionkey"), nsid)
 
-		res.Message = "Login successfully"
+		res.Message = string(ver)
```
Comment (veezhang) [inline controllers/db.go:62]: The business code does not provide new features, and the follow-up is only maintenance.
Let studio do this?

Comment (1uvu) [reply controllers/db.go:62]: ok.

**Comment by veezhang** [comment]
You need to rebase to resolve the conflict.

File: ccore/nebula/gateway/dao/dao.go (hunk 1 above)
Comment (1uvu) [reply ccore/nebula/gateway/dao/dao.go:279]: > How about just return `nebula.GraphClient, error`?

The Studio need the returned nsid to identify a connect session. So If return `GraphClient` interface may cause some problems.

File: controllers/db.go (hunk 2 above)
Comment (1uvu) [reply controllers/db.go:62]: > The business code does not provide new features, and the follow-up is only maintenance. Let studio do this?

But the Studio don't konw which nebula client it connected.

**Comment by 1uvu** [comment]
> You need to rebase to resolve the conflict.

Ok.

File: ccore/nebula/gateway/dao/dao.go (hunk 1 above)
Comment (veezhang) [reply ccore/nebula/gateway/dao/dao.go]: How about return `nsid, nebula.GraphClient, error` ? 
Or define an interface to return?

This is convenient for expansion, and if more information is needed in the future, more and more return values will be returned.

File: controllers/db.go (hunk 2 above)
Comment (veezhang) [reply controllers/db.go:61]: The `studio` is use golang server now, the api will no longer use the controller here.

**Review by veezhang — APPROVED** [review]
LGTM

**Pull request merged**